public/media_engagement/styles.css

```diff
@@ -21,11 +21,14 @@
     background-color: #B6D4E7;
     
 }
-.btn-logout{
+.txt-brown {
+    color: #8d3538; 
+}
+.btn-brown{
```

Comment:
```suggestion
.btn-brown {
```

Reply:
@DanielMwendwa please apply these changes ahead of requesting review - thanks!